apps/air-discount-scheme/backend/src/app/modules/flight/flight.service.ts

```diff
@@ -114,7 +114,6 @@ export class FlightService {
     return this.flightLegModel.findAll({
       where: {
         ...(body.airline ? { airline: body.airline } : {}),
-        ...(body.cooperation ? { cooperation: body.cooperation } : {}),
```

Comment:
I believe we still need this for legacy reasons. Although future flights won't have a cooperation, the previous ones did.

Reply:
If I am not mistaken, this and the things you dittoed are so that Vegagerðin can filter for the `Norlandair` airline and get all the flights that icelandair did in cooperation with Norlandair.

I know we can add the cooperation select box and use that. But we don't really need that feature for Vegagerðin because they only need to send to filter after the airlines that are in direct connection with them.

Reply:
I am essentially saying that it was an unused feature to filter for Norlandair airline and I don't see the point in adding the select box to be able to do it for legacy data.

Reply:
mm I see, good point!